src/cdk/bootstrap.ts

```diff
@@ -238,15 +238,16 @@ export class MiraBootstrap {
     const resultedEnv = this.env || env
     const q = process.platform === 'win32' ? '"' : '\''
     let appPath = path.resolve(__dirname, filename)
+
     if (fs.existsSync('node_modules/mira')) {
       if (fs.lstatSync('node_modules/mira/dist').isSymbolicLink()) {
         // Mira has been locally linked.
         try { fs.mkdirSync('node_modules/mira-bootstrap') } catch (e) {
           // NOOP
         }
-        fs.writeFileSync('node_modules/mira-bootstrap/bootstrap-app.js',
-          'require(\'mira/dist/src/cdk/app.js\')', 'utf8')
-        appPath = 'node_modules/mira-bootstrap/bootstrap-app.js'
+        fs.writeFileSync(`node_modules/mira-bootstrap/bootstrap-${filename}`,
+          `require('mira/dist/src/cdk/${filename}')`, 'utf8')
```

Comment:
Am curious regarding this: this is code specific for a local Mira link.  Was it intended to "fix" this feature?  Otherwise am not sure this is actually what would be desired?

Reply:
Yes, it was intended to fix the domain feature. Previously, it was always adding the `require('mira/dist/src/cdk/app.js` to the `bootstrap-app.js` file even when the `domain` command was invoked. With this change it creates a bootstrap file for the right command, ie adds the `app.js` require to the `bootstrap-app.js` file for the deploy command and the `domain.js` for the `domain` command.

Reply:
ok LGTM